pkg/controller/selector/selector.go

```diff
@@ -85,6 +85,7 @@ func (s *selector) SelectSupportedPoolForVolume(vol *model.VolumeSpec) (*model.S
 			filterRequest["id"] = vol.PoolId
 		}
 		filterRequest["storageType"] = prf.StorageType
+		filterRequest["status"] = "available"
```

Comment:
Do we know what all the possible values for poll status? Adding this filter limits to creation only if status is set as "Available"

Reply:
Asper discussion we have to have a standard enum for opensds pool status. as of now only available pools can use for volume creation